### DISCUSSION TITLE
envd biweekly newsletter

### DISCUSSION THREAD

**Discussion by lalawuu** [tensorchord/envd#1182] [Ideas]
Could we open a newsletter for envd users or watcher. 
I had some ideas for the content:
- For Students and ML/AI beginner: a kaggle problem or kaggle like ML/AI/Data Science Problem with its tutorial(Juypter Notebook).
- The updates and new user story of envd.
-  Something new about ML/AI/Data Science and MLOps

**Comment by gaocegege** [comment]
Some suggestions:

- For investors and other MLOps companies: Introduce one MLOps company or its product in the bi-weekly newsletter. About its business model, its open-sourced projects, what problem they want to solve.

**Comment by gaocegege** [comment]
Here is an example https://www.linkedin.com/pulse/issue-204-ml-engineer-alejandro-saucedo/

**Reply by gaocegege** [reply]
Here is another example https://mlopsroundup.substack.com/

**Comment by gaocegege** [comment]
Maybe we can use https://www.getrevue.co/

**Reply by kemingy** [reply]
I wouldn't say I like this one because:

- its tracker or something else is not HTTPS, which will be blocked if the user browser has strict checks (like me)
- owned by twitter, but twitter may not be stable, you know what happened

**Comment by lalawuu** [comment]
### Overview of the newsletter
- Publish every two weeks
- Target audience:
    - envd users and fans
    - AI/ML Engineers(Not Researcher)
    - Students and ML/AI beginner
    - Companies's Tech Offical and insteresting investors
- Goals:
    - **Most Important:** Gather MLOps or AI/ML engineering fans together and build a neighborhood for them.
    - Promote our influence and exposure in ML/AI community especially in MLOps area.
    - Become the top 1 newsletter in MLOps area

### What content we may publish

#### envd summary

A short essay describing the activities of 
- envd community meetings notes
- envd  development changelog
But it should be orginized to be a in-depth article to clearly state changes and thinking of envd.

#### MLOps Radar

Introduction of MLOps trending:
- new player or tech of MLOps
- important change of MLOps 
- new methodology/thinking of MLOps

The MLOps Radar is to give a whole a picture of what happended in last two weeks. It aimed at AI/ML Engineers, Companies's Tech Offical and insteresting investors. So it should't cover too much tech details but qualitative analysis matters.

#### Highlights for ML/AI

In this part, we will catch the best practices or engineering experiences in AI/ML area. such as:

- Synthetic Data Generation: How to Protect User Privacy
- Multi-cloud and cloud-agnostic computing practices

It is for AI/ML engineers to help them build better AI/ML applications. This part will have more details and dig into the core of tech.

#### envd camp

Intersting problems or challenges for students and beginners. We'd better choose some interesting contests such as kaggle or famous tools like xgboost. It worth considering because beginers have a strong demand and it also will bring a lot user search exposure. Here is some examples:

- Kaggle linear regression all-in-one solution
- 10mins to set up your stock prediction environment

We also can actively collobrate with other ML/AI/Data projects to make it sexy.

### How to organize the edition work

In order to create content regularly and at a high quality. We'd better choose a chef editor. Maintainers or contributors in envd community take turns to be the edior. 

The editors can call submissions from all envd community memebers.

#### Reward for Contributors

I think we could have some kind of reward for contributors to submit articles qualified.  A gift card and a meal?
 
### How we publish the newsletter

We can hold four ways to publish the newsletter

- substack: for common users who like beautiful UI and substack have the discussion function.
- medium/blog: search engine optimization
- Wechat: for chinese users

**Reply by gaocegege** [reply]
LGTM, let's give it a try!

**Comment by AlexXi19** [comment]
I personally would love to read MLOps or ML related articles from an industry standpoint. 

Genuine question, are there really any ML/AI fans or students who play around with models in their free time? If so, what are some of the things they play around with? Because I imagine it would be hard to play around with large deep learning models.

**Reply by lalawuu** [reply]
Thanks for you reply. I mentioned students because I spend a lot of time help student (non-cs) set up their environment when I was TA. We could prepare some basic projects for students.
As for large deep learning models, the only problem is the cuda equipment with the help of envd. I think we can solve it by providing a envd cloud service LOL in the future like colab or we could embed envd in colab service.

**Reply by AlexXi19** [reply]
Ah yes, that is indeed a use case I have thought about. 

> envd cloud service
😏

**Comment by gaocegege** [comment]
We need a name for the newsletter.

It's your turn to show up your ability.

**Reply by gaocegege** [reply]
Here are some ideas to get you started: 

- MLOps roundup from 
[@mlopsroundup](https://twitter.com/mlopsroundup)
 
- The ML Engineer

**Reply by kemingy** [reply]
How about "42ml"?

* 42 is a common random seed. Mainly because it's the "[Answer to the Ultimate Question of Life, the Universe, and Everything](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life,_the_Universe,_and_Everything_(42))"
* ml is short for machine learning

**Reply by xieydd** [reply]
How about "MLOops", o͝ops hh.
